install-scripts/install_ipfix-rita.sh

```diff
@@ -0,0 +1,503 @@
+	else
+	        #do something great here
+	        rita_conf="/etc/rita/config.yaml"
+	fi
+
+	db_root="IPFIX"
+	mongo_uri="mongodb://${rita_system#*@}:27017"
+	mongo_auth=`get_line $rita_conf "AuthenticationMechanism: "`
+	mongo_tls_enable=`get_line $rita_conf "Enable: "`
+	mongo_tls_cert_check=`get_line $rita_conf "VerifyCertificate: "`
+	mongo_tls_ca_path=`get_line $rita_conf "CAFile: "`
+
+	#If the config file was copied from a remote server, delete that file
+	if [ "$delete_conf" == "true" ]; then
+	        rm "./config.yaml"
+	fi
+
+	#If the config specified a CA file, let's copy that
+	if [[ $mongo_tls_ca_path != *"null" ]]; then
+		echo $mongo_tls_ca_path
+		ca_file=${mongo_tls_ca_path#"CAFile: "}
+		scp $rita_system:$ca_file $ca_file
+	fi
+
+	write_converter_conf $db_root $mongo_uri $mongo_auth $mongo_tls_enable $mongo_tls_cert_check $mongo_tls_ca_path
+} #End of get_rita_data_noninteractive
+
+write_converter_conf() {
+	RITA_DATASET_DBROOT=$1
+	RITA_MONGO_URI=$2
+	RITA_MONGO_AUTH=$3
+	RITA_MONGO_TLS=$4
+	RITA_MONGO_TLS_CHECK_CERT=$5
+	RITA_MONGO_TLS_CERT_PATH=$6
+
+awk -v db_root="$RITA_DATASET_DBROOT" \
+-v mongo_uri="$RITA_MONGO_URI" \
+-v mongo_auth="$RITA_MONGO_AUTH" \
+-v mongo_tls_enable="$RITA_MONGO_TLS" \
+-v mongo_tls_cert_check="$RITA_MONGO_TLS_CHECK_CERT" \
+-v mongo_tls_ca_path="$RITA_MONGO_TLS_CERT_PATH" '
+# flag is used to determine if we are in the right scope
+
+# Unset the flag if we see "abc:" or "  abc:" on a line
+# by itself if there are 2 or less preceding spaces
+/^(  )?[^ ]+:$/{
+  flag=""
+}
+
+# Trigger the flag as we are entering the scope
+/  RITA-MongoDB:/{
+  flag=1
+}
+
+flag && NF && /ConnectionString:/{
+  match($0,/^ +/);
+  val=substr($0,RSTART,RLENGTH);
+  $NF=mongo_uri;
+  print val $0;
+  next
+}
+
+flag && NF && /AuthenticationMechanism:/{
+  match($0,/^ +/);
+  val=substr($0,RSTART,RLENGTH);
+  $NF=mongo_auth;
+  print val $0;
+  next
+}
+
+flag && NF && /Enable:/{
+  match($0,/^ +/);
+  val=substr($0,RSTART,RLENGTH);
+  $NF=mongo_tls_enable;
+  print val $0;
+  next
+}
+
+flag && NF && /VerifyCertificate:/{
+  match($0,/^ +/);
+  val=substr($0,RSTART,RLENGTH);
+  $NF=mongo_tls_cert_check;
+  print val $0;
+  next
+}
+
+flag && NF && /CAFile:/{
+  match($0,/^ +/);
+  val=substr($0,RSTART,RLENGTH);
+  $NF=mongo_tls_ca_path;
+  print val $0;
+  next
+}
+
+flag && NF && /DBRoot:/{
+  match($0,/^ +/);
+  val=substr($0,RSTART,RLENGTH);
+  $NF=db_root;
+  print val $0;
+  next
+}
+
+1
+' $INSTALLATION_ETC_DIR/converter/converter.yaml > $INSTALLATION_ETC_DIR/converter/converter-new.yaml && \
+	mv $INSTALLATION_ETC_DIR/converter/converter-new.yaml $INSTALLATION_ETC_DIR/converter/converter.yaml
+} #End of write_converter_conf
+
+# Stop if there are any errors
+set -e
+# Change dir to script dir
+echo "tyring to cd to $(dirname "$BASH_SOURCE[0]")"
+_OLD_DIR=$(pwd); cd "$(dirname "$BASH_SOURCE[0]")";
+
+if [[ $EUID -ne 0 ]]; then
+   echo "This script must be run with administrator privileges."
```

Comment:
Looks like we're missing `>&2` from here to the bottom of the file. (Mainly at the bottom)